src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts

```diff
@@ -1,6 +1,6 @@
 import { ChangeDetectionStrategy, Component, Input, OnInit } from '@angular/core';
 import { FormNode } from '@forms/services/dynamic-form.types';
-import { Brakes as BrakesTemplate } from '@forms/templates/hgv/hgv-brakes.template';
```

Comment:
Should this file be deleted as well if HGVs don't display tyres?

Reply:
It has?